### PR TITLE
add provides for pulpcore-plugin()

### PR DESCRIPTION
**Pull request by evgeni**
and obsolete old (python 3.6) names

### CODE REVIEW

File: packages/python-pulp-certguard/python-pulp-certguard.spec [hunk 1]
```diff
@@ -31,6 +31,8 @@ Requires:       %{?scl_prefix}python%{python3_pkgversion}-pyOpenSSL
 Requires:       %{?scl_prefix}python%{python3_pkgversion}-pulpcore >= 3.10
 Requires:       %{?scl_prefix}python%{python3_pkgversion}-setuptools
 
+Provides:       pulpcore-plugin(certguard) = %{veru}
```
Comment (ekohl) [inline packages/python-pulp-certguard/python-pulp-certguard.spec]:
```suggestion
Provides:       pulpcore-plugin(certguard) = %{version}
```

**Review by ekohl — COMMENTED** [review]
:+1: for the general direction. Looks like there's one typo though.

File: packages/python-pulp-certguard/python-pulp-certguard.spec (hunk 1 above)
Comment (evgeni) [reply packages/python-pulp-certguard/python-pulp-certguard.spec]: lol, vim, what did you do?!

**Review by ekohl — COMMENTED** [review]
Can we have this in 3.14 too so we can start to change the installer?

Thinking more about this: I wonder if the installer can ensure `pulpcore-plugin(galaxy-ng)` to be present or if Puppet can't do that. That'd be a good thing to verify.

**Comment by evgeni** [comment]
Yes, I intend to backport that to 3.14 once everyone agreed on the color of the bikeshed.

I think modern Puppet understands provides.

**Comment by ekohl** [comment]
I tried:
```puppet
package { 'tfm-rubygem(foreman_puppet)':
  ensure => installed,
}
```

Then this results in:
```console
# puppet --version
7.11.0
# puppet apply --noop --debug test.pp 
....
Info: Applying configuration version '1634568878'
Debug: Prefetching yum resources for package
Debug: Executing: '/bin/rpm --version'
Debug: Executing '/bin/rpm -qa --nosignature --nodigest --qf '%{NAME} %|EPOCH?{%{EPOCH}}:{0}| %{VERSION} %{RELEASE} %{ARCH}\n' | sort'
Debug: Executing: '/bin/rpm -q tfm-rubygem(foreman_puppet) --nosignature --nodigest --qf %{NAME} %|EPOCH?{%{EPOCH}}:{0}| %{VERSION} %{RELEASE} %{ARCH}\n'
Debug: Finishing transaction 12300
Debug: Storing state
Debug: Pruned old state cache entries in 0.00 seconds
Debug: Stored state in 0.21 seconds
Notice: Applied catalog in 0.92 seconds
```

Looks like it uses `rpm -q $PACKAGE` so I think that confirms it should work.

**Review by ekohl — APPROVED** [review]

**Pull request merged**